### ISSUE TITLE
Again a problem with Topaz

### ISSUE THREAD

**Issue by MariusMatutiae** [apprenticeharper/DeDRM_tools#117]
I have an oldish (2010) book in Topaz format, which will not import to HTMLZ, I only see it in AZW format.  Can you help me?

The import log, having started Calibre in debug mode, is the following:

```
WARNING: Failed to set default libc locale, using en_US.UTF-8
calibre Debug log
calibre 2.63  embedded-python: True is64bit: True
Linux-4.2.0-42-generic-x86_64-with-debian-jessie-sid Linux ('64bit', 'ELF')
('Linux', '4.2.0-42-generic', '#49~14.04.1-Ubuntu SMP Wed Jun 29 20:22:11 UTC 2016')
Python 2.7.9
Linux: ('debian', 'jessie/sid', '')
Successfully initialized third party plugins: DeDRM (6, 4, 3)

(process:5442): Gtk-WARNING **: Locale not supported by C library.
	Using the fallback 'C' locale.
Starting up...
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKkr-Thin.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKkr-Thin.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKsc-Black.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKsc-Black.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKkr-Black.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKkr-Black.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKjp-DemiLight.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKjp-DemiLight.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKjp-Light.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKjp-Light.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKsc-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKsc-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKtc-Light.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKtc-Light.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKtc-Medium.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKtc-Medium.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKjp-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKjp-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKkr-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKkr-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKjp-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKjp-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKsc-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKsc-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKjp-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKjp-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKkr-Light.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKkr-Light.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKtc-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKtc-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKsc-Light.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKsc-Light.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKtc-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKtc-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKsc-Thin.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKsc-Thin.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKsc-DemiLight.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKsc-DemiLight.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKtc-DemiLight.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKtc-DemiLight.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKsc-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKsc-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKsc-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKsc-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKkr-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKkr-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKjp-Thin.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKjp-Thin.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKkr-Medium.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKkr-Medium.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKkr-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKkr-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKtc-Black.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKtc-Black.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKjp-Black.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKjp-Black.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKsc-Medium.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKsc-Medium.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansMonoCJKtc-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansMonoCJKtc-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKtc-Thin.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKtc-Thin.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKtc-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKtc-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKjp-Bold.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKjp-Bold.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKkr-Regular.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKkr-Regular.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKkr-DemiLight.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKkr-DemiLight.otf'
Failed to read metadata from font file: /usr/share/fonts/opentype/noto/NotoSansCJKjp-Medium.otf [Errno 13] Permission denied: u'/usr/share/fonts/opentype/noto/NotoSansCJKjp-Medium.otf'
WARNING: Failed to set default libc locale, using en_US.UTF-8

Found device nodes: /dev/sdb1 None None
Job: 1 Get device information started
DeviceJob: 1 Get device information done, calling callback
DeviceJob: 1 Get device information callback returned
Job: 2 Set library information started
DeviceJob: 2 Set library information done, calling callback
DeviceJob: 2 Set library information callback returned
Job: 3 Get list of books on device started
DEBUG:    0.0 Job:USBMS: Fetching list of books from device. Device=  KINDLE21 Get device information  oncard= finishedNone

No details available.DEBUG:    0.0
 USBMS: dirs are:Job: /media/ME/Kindle/  2 documentsSet library information
 finished
No details available.
Started up in 7.76 seconds with 383 books
DEBUG:    1.0 USBMS: scan from root False /media/ME/Kindle/documents
DEBUG:    1.1 USBMS: count found in cache: 406, count of files in metadata: 406, need_sync: False
DEBUG:    1.1 USBMS: Finished fetching list of books from device. oncard= None
DEBUG:    1.1 USBMS: Fetching list of books from device. Device= KINDLE2 oncard= carda
DEBUG:    1.1 USBMS: Fetching list of books from device. Device= KINDLE2 oncard= cardb
DeviceJob: 3 Get list of books on device done, calling callback
DeviceJob: metadata_downloaded: Starting set_books_in_library
DeviceJob: set_books_in_library: books to process= 406
DeviceJob: set_books_in_library finished: time= 0.0153980255127
DeviceJob: metadata_downloaded: updating views
DeviceJob: metadata_downloaded: syncing
DeviceJob: metadata_downloaded: refreshing ondevice
DeviceJob: metadata_downloaded: sending metadata_available signal
DeviceJob: 3 Get list of books on device callback returned
Job: 3 Get list of books on device finished
No details available.
DeDRM v6.4.3: Trying to decrypt B000PY4IDG_EBOK.azw
Using Library AlfCrypto DLL/DYLIB/SO
Decrypting Topaz ebook: The Dawn of Human Culture
Found 10 keys to try after 0.0 seconds
Trying: 6236NNNX
Trying: 835bXnVc
Book Key Found! (b24d2edc83518ec3)
Processing Section: img
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .  
Processing Section: glyphs
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .  
Processing Section: dict
. . . .  
Processing Section: other
. . . .  
Processing Section: page
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .  
Processing Section: metadata
. . . .  
Successfully Extracted Topaz contents
Updating to color images if available
Creating cover.jpg
Processing Dictionary
Processing Meta Data and creating OPF
Processing StyleSheet
Unknown Token: median_d
DeDRM v6.4.3: Failed to decrypt with error: list index out of range
DeDRM v6.4.3: Looking for new default Kindle Key after 0.3 seconds
DeDRM v6.4.3: Running kindlekey.py under Wine
DeDRM v6.4.3: Command line: “ wine python.exe "/home/ME/.config/calibre/plugins/DeDRM/libraryfiles/kindlekey.py" "/home/ME/.config/calibre/plugins/DeDRM/libraryfiles/winekeysdir"”
kindlekey.py v2.2
Copyright © 2010-2016 by some_updates, Apprentice Alf and Apprentice Harper
searching for kinfoFiles in C:\users\ME\Local Settings\Application Data
Found K4PC 1.9+ kinf2011 file: C:\users\ME\Local Settings\Application Data\Amazon\Kindle\storage\.kinf2011
Decrypted key file using IDString '0' and UserName '6d6172696f'
Saved a key to Z:\home\ME\.config\calibre\plugins\DeDRM\libraryfiles\winekeysdir\kindlekey1.k4i
DeDRM v6.4.3: Found and decrypted 1 key file
DeDRM v6.4.3: Found 1 new key
Decrypting Topaz ebook: The Dawn of Human Culture
Found 4 keys to try after 0.6 seconds
Trying: 6236NNNX
Trying: 835bXnVc
Book Key Found! (b24d2edc83518ec3)
Processing Section: img
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .  
Processing Section: glyphs
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .WARNING: Failed to set default libc locale, using en_US.UTF-8
 . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .  
Processing Section: dict
. . . .  
Processing Section: other
. . . .  
Processing Section: page
. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .  
Processing Section: metadata
. . . .  
Successfully Extracted Topaz contents
Updating to color images if available
Creating cover.jpg
Processing Dictionary
Processing Meta Data and creating OPF
Processing StyleSheet
Unknown Token: median_d
DeDRM v6.4.3: Ultimately failed to decrypt after 0.9 seconds.
Read the FAQs at Alf's blog: http://apprenticealf.wordpress.com/
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "site-packages/calibre/customize/ui.py", line 154, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 604, in run
  File "calibre_plugins.dedrm.__init__", line 554, in KindleMobiDecrypt
DeDRMError: DeDRM v6.4.3: Ultimately failed to decrypt after 0.9 seconds
Added The Dawn of Human Culture to db in: 0.1
Added 1 books in 1.5 seconds
Worker Launch took: 0.255874872208
Job: 4 Convert book 1 of 1 (The Dawn of Human Culture) finished
WARNING: Failed to set default libc locale, using en_US.UTF-8
	Convert book 1 of 1 (The Dawn of Human Culture)
	Resolved conversion options
	calibre version: 2.63.0
	{'asciiize': False,
	 'author_sort': None,
	 'authors': None,
	 'base_font_size': 0.0,
	 'book_producer': None,
	 'change_justification': u'original',
	 'chapter': u"//*[((name()='h1' or name()='h2') and re:test(., '\\s*((chapter|book|section|part)\\s+)|((prolog|prologue|epilogue)(\\s+|$))', 'i')) or @class = 'chapter']",
	 'chapter_mark': u'pagebreak',
	 'comments': None,
	 'cover': None,
	 'debug_pipeline': None,
	 'dehyphenate': True,
	 'delete_blank_paragraphs': True,
	 'disable_font_rescaling': False,
	 'dont_compress': False,
	 'duplicate_links_in_toc': False,
	 'embed_all_fonts': False,
	 'embed_font_family': None,
	 'enable_heuristics': False,
	 'expand_css': False,
	 'extra_css': None,
	 'extract_to': None,
	 'filter_css': u'',
	 'fix_indents': True,
	 'font_size_mapping': None,
	 'format_scene_breaks': True,
	 'html_unwrap_factor': 0.4,
	 'input_encoding': None,
	 'input_profile': <calibre.customize.profiles.InputProfile object at 0x7faf0ef38750>,
	 'insert_blank_line': False,
	 'insert_blank_line_size': 0.5,
	 'insert_metadata': False,
	 'isbn': None,
	 'italicize_common_cases': True,
	 'keep_ligatures': False,
	 'language': None,
	 'level1_toc': None,
	 'level2_toc': None,
	 'level3_toc': None,
	 'line_height': 0.0,
	 'linearize_tables': False,
	 'margin_bottom': 5.0,
	 'margin_left': 5.0,
	 'margin_right': 5.0,
	 'margin_top': 5.0,
	 'markup_chapter_headings': True,
	 'max_toc_links': 50,
	 'minimum_line_height': 120.0,
	 'mobi_file_type': u'old',
	 'mobi_ignore_margins': False,
	 'mobi_keep_original_images': False,
	 'mobi_toc_at_start': False,
	 'no_chapters_in_toc': False,
	 'no_inline_navbars': True,
	 'no_inline_toc': False,
	 'output_profile': <calibre.customize.profiles.KindlePaperWhiteOutput object at 0x7faf0ef38ed0>,
	 'page_breaks_before': u"//*[name()='h1' or name()='h2']",
	 'personal_doc': u'[PDOC]',
	 'prefer_author_sort': False,
	 'prefer_metadata_cover': False,
	 'pretty_print': False,
	 'pubdate': None,
	 'publisher': None,
	 'rating': None,
	 'read_metadata_from_opf': u'/tmp/calibre_2.63.0_tmp_uSgqzK/J5AjgY.opf',
	 'remove_fake_margins': True,
	 'remove_first_image': False,
	 'remove_paragraph_spacing': False,
	 'remove_paragraph_spacing_indent_size': 1.5,
	 'renumber_headings': True,
	 'replace_scene_breaks': u'',
	 'search_replace': '[]',
	 'series': None,
	 'series_index': None,
	 'share_not_sync': False,
	 'smarten_punctuation': False,
	 'sr1_replace': None,
	 'sr1_search': None,
	 'sr2_replace': None,
	 'sr2_search': None,
	 'sr3_replace': None,
	 'sr3_search': None,
	 'start_reading_at': None,
	 'subset_embedded_fonts': False,
	 'tags': None,
	 'timestamp': None,
	 'title': None,
	 'title_sort': None,
	 'toc_filter': None,
	 'toc_threshold': 6,
	 'toc_title': None,
	 'transform_css_rules': '[]',
	 'unsmarten_punctuation': False,
	 'unwrap_lines': True,
	 'use_auto_toc': False,
	 'verbose': 2}
	InputFormatPlugin: MOBI Input running
	on /tmp/calibre_2.63.0_tmp_uSgqzK/eYqkTJ.azw
	Traceback (most recent call last):
	  File "site.py", line 66, in main
	  File "site-packages/calibre/utils/ipc/worker.py", line 190, in main
	  File "site-packages/calibre/gui2/convert/gui_conversion.py", line 31, in gui_convert_override
	  File "site-packages/calibre/gui2/convert/gui_conversion.py", line 25, in gui_convert
	  File "site-packages/calibre/ebooks/conversion/plumber.py", line 1060, in run
	  File "site-packages/calibre/customize/conversion.py", line 241, in __call__
	  File "site-packages/calibre/ebooks/conversion/plugins/mobi_input.py", line 33, in convert
	  File "site-packages/calibre/ebooks/mobi/reader/mobi6.py", line 71, in __init__
	TopazError: This is an Amazon Topaz book. It cannot be processed.

WARNING: Failed to set default libc locale, using en_US.UTF-8
Convert book 1 of 1 (The Dawn of Human Culture)
Resolved conversion options
calibre version: 2.63.0
{'asciiize': False,
 'author_sort': None,
 'authors': None,
 'base_font_size': 0.0,
 'book_producer': None,
 'change_justification': u'original',
 'chapter': u"//*[((name()='h1' or name()='h2') and re:test(., '\\s*((chapter|book|section|part)\\s+)|((prolog|prologue|epilogue)(\\s+|$))', 'i')) or @class = 'chapter']",
 'chapter_mark': u'pagebreak',
 'comments': None,
 'cover': None,
 'debug_pipeline': None,
 'dehyphenate': True,
 'delete_blank_paragraphs': True,
 'disable_font_rescaling': False,
 'dont_compress': False,
 'duplicate_links_in_toc': False,
 'embed_all_fonts': False,
 'embed_font_family': None,
 'enable_heuristics': False,
 'expand_css': False,
 'extra_css': None,
 'extract_to': None,
 'filter_css': u'',
 'fix_indents': True,
 'font_size_mapping': None,
 'format_scene_breaks': True,
 'html_unwrap_factor': 0.4,
 'input_encoding': None,
 'input_profile': <calibre.customize.profiles.InputProfile object at 0x7faf0ef38750>,
 'insert_blank_line': False,
 'insert_blank_line_size': 0.5,
 'insert_metadata': False,
 'isbn': None,
 'italicize_common_cases': True,
 'keep_ligatures': False,
 'language': None,
 'level1_toc': None,
 'level2_toc': None,
 'level3_toc': None,
 'line_height': 0.0,
 'linearize_tables': False,
 'margin_bottom': 5.0,
 'margin_left': 5.0,
 'margin_right': 5.0,
 'margin_top': 5.0,
 'markup_chapter_headings': True,
 'max_toc_links': 50,
 'minimum_line_height': 120.0,
 'mobi_file_type': u'old',
 'mobi_ignore_margins': False,
 'mobi_keep_original_images': False,
 'mobi_toc_at_start': False,
 'no_chapters_in_toc': False,
 'no_inline_navbars': True,
 'no_inline_toc': False,
 'output_profile': <calibre.customize.profiles.KindlePaperWhiteOutput object at 0x7faf0ef38ed0>,
 'page_breaks_before': u"//*[name()='h1' or name()='h2']",
 'personal_doc': u'[PDOC]',
 'prefer_author_sort': False,
 'prefer_metadata_cover': False,
 'pretty_print': False,
 'pubdate': None,
 'publisher': None,
 'rating': None,
 'read_metadata_from_opf': u'/tmp/calibre_2.63.0_tmp_uSgqzK/J5AjgY.opf',
 'remove_fake_margins': True,
 'remove_first_image': False,
 'remove_paragraph_spacing': False,
 'remove_paragraph_spacing_indent_size': 1.5,
 'renumber_headings': True,
 'replace_scene_breaks': u'',
 'search_replace': '[]',
 'series': None,
 'series_index': None,
 'share_not_sync': False,
 'smarten_punctuation': False,
 'sr1_replace': None,
 'sr1_search': None,
 'sr2_replace': None,
 'sr2_search': None,
 'sr3_replace': None,
 'sr3_search': None,
 'start_reading_at': None,
 'subset_embedded_fonts': False,
 'tags': None,
 'timestamp': None,
 'title': None,
 'title_sort': None,
 'toc_filter': None,
 'toc_threshold': 6,
 'toc_title': None,
 'transform_css_rules': '[]',
 'unsmarten_punctuation': False,
 'unwrap_lines': True,
 'use_auto_toc': False,
 'verbose': 2}
InputFormatPlugin: MOBI Input running
on /tmp/calibre_2.63.0_tmp_uSgqzK/eYqkTJ.azw
Traceback (most recent call last):
  File "site.py", line 66, in main
  File "site-packages/calibre/utils/ipc/worker.py", line 190, in main
  File "site-packages/calibre/gui2/convert/gui_conversion.py", line 31, in gui_convert_override
  File "site-packages/calibre/gui2/convert/gui_conversion.py", line 25, in gui_convert
  File "site-packages/calibre/ebooks/conversion/plumber.py", line 1060, in run
  File "site-packages/calibre/customize/conversion.py", line 241, in __call__
  File "site-packages/calibre/ebooks/conversion/plugins/mobi_input.py", line 33, in convert
  File "site-packages/calibre/ebooks/mobi/reader/mobi6.py", line 71, in __init__
TopazError: This is an Amazon Topaz book. It cannot be processed.


```

**Comment by apprenticeharper** [comment]
You've found a Topaz book that contains coding that we don't know about. Please post the encrypted book file somewhere I can download it. I think you can attach it to a message here. Thanks.

**Comment by MariusMatutiae** [comment]
Wow, a rarity, how lucky of me. Well, at least I will be contributing to the general progress. Here you go. 

https://app.box.com/s/8whwvuysam5vadve1ts2vjsxlq8wmdhw

**Comment by MariusMatutiae** [comment]
Wow, a rarity, how lucky of me.

Well, at least I will be contributing to the general progress. Here you go.

Thanks a lot for taking the time to look into it.


https://app.box.com/s/8whwvuysam5vadve1ts2vjsxlq8wmdhw

**Comment by apprenticeharper** [comment]
Thanks, I have downloaded the ebook now. I'll see what I can do.

**Comment by apprenticeharper** [comment]
This problem should be fixed by v6.5.0 that I have just uploaded.

https://github.com/apprenticeharper/DeDRM_tools/releases/tag/v6.5.0

**Comment by MariusMatutiae** [comment]
Indeed it is fixed: it works flawlessly. Thank you for your hard work, I really appreciate.
